SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -1655,7 +1654,7 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::ApplyTps(int totalNum)
         TransformNOutput(tps, crestSpokes, crestOutputFile);
 
     }
-
+    return 0;
```

Comment:
If the return is always 0, make the function void instead of int and remove the return.